Documentation/Storage-Configuration/Ceph-CSI/ceph-csi-drivers.md

```diff
@@ -107,52 +109,51 @@ apiVersion: v1
                 storage: 1Gi
 ```
 
-A volume claim template is defined inside the pod spec which refers to a volume
-provisioned and used by the pod with its lifecycle. The volumes are provisioned
-when pod get spawned and destroyed at time of pod delete.
+A volume claim template is defined inside the pod spec, and defines a volume to be
+provisioned and used by the pod within its lifecycle. Volumes are provisioned
+when a pod is spawned and destroyed when the pod is deleted.
+
+Refer to the [ephemeral-doc](https://kubernetes.io/docs/concepts/storage/ephemeral-volumes/#generic-ephemeral-volumes) for more info.
+See example manifests for an [RBD ephemeral volume](https://github.com/rook/rook/tree/master/deploy/examples/csi/rbd/pod-ephemeral.yaml)
+and a [CephFS ephemeral volume](https://github.com/rook/rook/tree/master/deploy/examples/csi/cephfs/pod-ephemeral.yaml).
 
-Refer to [ephemeral-doc](https://kubernetes.io/docs/concepts/storage/ephemeral-volumes/#generic-ephemeral-volumes) for more info.
-Also, See the example manifests for an [RBD ephemeral volume](https://github.com/rook/rook/tree/master/deploy/examples/csi/rbd/pod-ephemeral.yaml) and a [CephFS ephemeral volume](https://github.com/rook/rook/tree/master/deploy/examples/csi/cephfs/pod-ephemeral.yaml).
 
 ## CSI-Addons Controller
 
-The CSI-Addons Controller handles the requests from users to initiate an operation. Users create a CR that the controller inspects, and forwards a request to one or more CSI-Addons side-cars for execution.
+The CSI-Addons Controller handles requests from users. Users create a CR
+that the controller inspects and forwards to one or more CSI-Addons sidecars for execution.
 
 ### Deploying the controller
 
-Users can deploy the controller by running the following commands:
+Administrators may deploy the controller by running the following commands:
```

Comment:
Instead of mentioning users, our new preference is to use the command form. Not all Rook docs are following this, but that's the goal.
```suggestion
Deploy the controller by running the following commands:
```